src/coreclr/inc/yieldprocessornormalized.h

```diff
@@ -175,7 +221,35 @@ FORCEINLINE void YieldProcessorNormalizedForPreSkylakeCount(
 //     }
 FORCEINLINE void YieldProcessorNormalizedForPreSkylakeCount(unsigned int preSkylakeCount)
 {
-    YieldProcessorNormalizedForPreSkylakeCount(YieldProcessorNormalizationInfo(), preSkylakeCount);
+    // This function does not forward to the one above because it is used by some code under utilcode, where
+    // YieldProcessorNormalizationInfo cannot be used since normalization does not happen in some of its consumers. So this
+    // version uses the fields in YieldProcessorNormalization directly.
+
+    _ASSERTE(preSkylakeCount != 0);
+
+    if (sizeof(SIZE_T) <= sizeof(unsigned int))
+    {
+        // On platforms with a small SIZE_T, prevent overflow on the multiply below
+        const unsigned int MaxCount = UINT_MAX / YieldProcessorNormalization::MaxYieldsPerNormalizedYield;
```

Comment:
why is this being computed multiple times, can this just be defined globally?

Reply:
I prefer the locality of relevance of this computation to the code below it over refactoring duplicates in this case. Amongst other reasons, the reason for duplicating this code is explained in the comment above. There are probably many more ways it could be refactored to minimize duplication, it's just not worthy of refactoring.